modules/data/generic_pdf.components.json

```diff
@@ -37,5 +37,11 @@
         "selectorData": "download",
         "strategy": "id",
         "groups": []
+    },
+
+    "zipcode-field": {
+        "selectorData": "pdfjs_internal_id_622R",
```

Comment:
I don't necessarily trust PDFjs internal CSS ids, I think a better selector would be `input[name='ZIP Code']` with strategy as `css`.